index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React</title>
+    <title>React redux capstone</title>
```

Comment:
Please make sure to use a descriptive title to give a clear idea about the project react redux capstone doens't reflect what your project does or contains